spotify_player/Cargo.toml

```diff
@@ -11,6 +11,7 @@ readme = "../README.md"
 
 [dependencies]
 anyhow = "1.0.86"
+blake3 = "1.5.3"
```

Comment:
no need to use a 3rd party library, rust has https://doc.rust-lang.org/std/hash/index.html